examples/conways_game_of_life/conways_game_of_life/server.py

```diff
@@ -1,12 +1,32 @@
-from mesa.visualization.modules import CanvasGrid
-from mesa.visualization.ModularVisualization import ModularServer
+from mesa.visualization.VegaVisualization import VegaServer
+from mesa.visualization.UserParam import UserSettableParameter
 
-from .portrayal import portrayCell
-from .model import ConwaysGameOfLife
 
+from .model import ConwaysGameOfLife
 
-# Make a world that is 50x50, on a 250x250 display.
-canvas_element = CanvasGrid(portrayCell, 50, 50, 250, 250)
+grid_spec = """
+{
+    "$schema": "https://vega.github.io/schema/vega-lite/v3.json",
+    "width": 500,
+    "height": 500,
+    "data": {"name": "agents"},
+    "mark": "bar",
+    "encoding": {
+      "x": {"type": "nominal", "field": "x"},
+      "y": {"type": "nominal", "field": "y"},
+      "color": {"type": "nominal", "field": "isAlive"}
+    }
+}
+"""
 
-server = ModularServer(ConwaysGameOfLife, [canvas_element], "Game of Life",
-                       {"height": 50, "width": 50})
+server = VegaServer(
```

Comment:
Because it says - `VegaServer`, are you suggesting that we maintain both or throw away the ModularServer?

Reply:
There was some discussion on this somewhere else and the idea is to maintain both for now. While VegaServer adds several new functionalities, it can't handle custom visualisation elements and would break every existing model